tests/org.eclipse.swt.tests.win32/JUnit Tests/org/eclipse/swt/tests/win32/Win32DPIUtilTests.java

```diff
@@ -146,15 +147,15 @@ public void scaleUpInteger() {
 		int valueAt150 = 8;
 		int valueAt100 = 5;
 
-		int scaledValue = Win32DPIUtils.pointToPixel(valueAt100, 200);
```

Comment:
Shouldn't the test be moved to `DPIUtilTests` instead of adapting the referenced class here?